gulpfile.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const gulp = require('gulp');
+const boilerplate = require('appium-gulp-plugins').boilerplate.use(gulp);
+const { fs } = require('appium-support');
+
+
+boilerplate({
+  build: 'appium-webdriveragent',
+  projectRoot: __dirname,
+});
+
+
+gulp.task('clean:carthage', function cleanCarthage () {
+  return fs.rimraf('Carthage');
+});
+
+gulp.task('install:dependencies', gulp.series('transpile', 'clean:carthage', function installDependencies () {
```

Comment:
we also usually execute xcodebuild with clean option to make sure the derived data caches are empty after WDA update

Reply:
I presume there is a bunch of stuff from `appium-xcuitest-driver` that ought to exist in this package. When wiring it in that will come out.

Reply:
I don't have a full plan of what exactly should be moved out of xcuitest driver yet, but I believe we can discuss that later

Reply:
Me too.